docs/src/pages/components/geo/react.mdx

```diff
@@ -1,8 +1,3 @@
----
-title: Geo
```

Comment:
The title and description is in `index.page.mdx` and adding it here creates a duplicate